app/resources/Base.lproj/MainMenu.xib

```diff
@@ -673,6 +673,6 @@
         <customObject id="733" customClass="OpenDriveWindowController"/>
         <customObject id="680" customClass="ChooseStringEncodingWindowController"/>
         <customObject id="744" customClass="SUUpdater"/>
-        <customObject id="Zzl-4N-uHo" userLabel="CLIController" customClass="CLIController"/>
+        <customObject id="Zzl-4N-uHo" userLabel="CLIController" customClass="CLIController" customModule="Hex_Fiend" customModuleProvider="target"/>
```

Comment:
I'm not exactly sure why, but this is necessary for validateMenuItem to get called.